scripts/create-aws-resources.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from os import environ
+
+from django.conf import settings
+import boto3
+import botocore
+
+AWS_ACCESS_KEY_ID = environ.get('AWS_ACCESS_KEY_ID')
```

Comment:
These should not be necessary because we are copying the VM host's AWS credentials into the VM. I think all we need to provide is the AWS `profile_name` to the `Session` constructor, which used to default to `rf-dev` in this project.